week1/project/ecommerce/src/components/Category.jsx

```diff
@@ -0,0 +1,7 @@
+function Category  ({data, className, onClick})  {
+    return (
+        <div className={className} onClick={() => onClick(data)}>{data}</div>
```

Comment:
also here you always use className `category-item` so you can add it here like that 
```js
<div className={`category-item ${className}`} onClick={() => onClick(data)}>{data}</div>
```
then it will be easy from anywhere to only add the variable class name/s

Reply:
fixed